docs/HTTP-batchsink.md

```diff
@@ -14,6 +14,22 @@ Properties
 
 **batchSize:** Batch size. Defaults to 1. (Macro enabled)
 
+**Write JSON As Array:** Whether to write the JSON as an array. Defaults to false. Ignored if batch size is 1.
+(Macro enabled)
+
+When set to true, the payload will be written as an array of JSON objects.
+Example - If batch size is 2, then the payload will be `[{"key":"val"}, {"key":"val"}]`
+
+When false, the payload will be JSON objects separated by a delimiter.
+Example - If batch size is 2, delimiter is "\n" , then the payload will be `{"key":"val"}\n{"key":"val"}`
+
+**Json Batch Key** Optional key to be used for wrapping json array as object " +
+"when batch size is more than 1. Leave empty for no wrapping of the array (Macro Enabled)
```

Comment:
put (Macro Enabled) at the end of the description on the next line.

Similar to above, it's simpler if this setting always applies regardless of the batch size.

Reply:
Updated